job-templates/kubernetes_release_staging_serial.json

```diff
@@ -15,7 +15,11 @@
           "--feature-gates": "KubeletPodResources=false"
         },
         "apiServerConfig": {
-          "--runtime-config": "extensions/v1beta1/podsecuritypolicies=true"
+          "--runtime-config": "extensions/v1beta1/podsecuritypolicies=true",
```

Comment:
can we alphabetize these so they are the same order in each file? (i know this is pretty nit picky ;( )

Reply:
done, though most of the items in these jsons are out of order, I updated a few of them